README.md

```diff
@@ -72,7 +72,26 @@ automatically scanning through your projects views for page templates.
 
 ### Setup
 
-1. Copy the template spec below into your project
+The accessibility test by default will live under the ``a11y`` folder in the project root directory, you may need to create
```

Comment:
@kyle-bowden just spotted, these should be single ` rather than with double ``

Reply:
@oscarduignan Thanks, just updated, going to wait for the 
**scalatest-accessibility-linter** to build latest version, seeing as it is merged in and then I will update the version in this PR.